profiles/models.py

```diff
@@ -22,4 +22,5 @@ def create_user_profile(sender, instance, created, **kwargs):
 
 post_save.connect(create_user_profile, sender=User)
 
-#User.profile = property(lambda u: u.profile())
+
+User.profile = property(lambda u: u.userprofile)
```

Comment:
ok i will make those changes and open new PR

On Wed, Jul 27, 2016 at 3:18 PM, Shiva Krishna <notifications@github.com>
wrote:

> In profiles/models.py
> <https://github.com/agiliq/fundraiser/pull/49#discussion_r72410881>:
>
> > @@ -22,4 +22,5 @@ def create_user_profile(sender, instance, created, **kwargs):
> >
> >  post_save.connect(create_user_profile, sender=User)
> >
> > -#User.profile = property(lambda u: u.profile())
> > +
> > +User.profile = property(lambda u: u.userprofile)
>
> @febinstephen <https://github.com/febinstephen> This line was initially
> commented and not sure why it was commented now, can you please recheck it
> and let me know what it does ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/agiliq/fundraiser/pull/49/files/1d6a3fb097abdfb3efd4a77bc89e64984f5540cb..c25eda319475b9e3dd335fe5edf8971153e04d5a#r72410881>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AG2pDzVanFr8nOUrCBbaMuzcNVTNbnRJks5qZylZgaJpZM4JREPw>
> .
>